### PR TITLE
fix(generate): improve error handling on generate command

### PR DESCRIPTION
**Pull request by jcvalerio**
Deliver better errors than `Unexpected token } in JSON at position X`
Fix issue #1047

### CODE REVIEW

File: packages/workspace/src/schematics/remove/lib/update-tsconfig.ts [hunk 1]
```diff
@@ -23,7 +23,12 @@ export function updateTsconfig(schema: Schema) {
 
         const tsConfigPath = 'tsconfig.json';
         if (tree.exists(tsConfigPath)) {
-          let contents = JSON.parse(tree.read(tsConfigPath).toString('utf-8'));
+          let contents = tree.read(tsConfigPath).toString('utf-8');
```
Comment (vsavkin) [inline packages/workspace/src/schematics/remove/lib/update-tsconfig.ts:26]: Could you instead readJsonInTree? It throws a good error and also strips comments

**Comment by vsavkin** [comment]
Thakn you for your contribution! Left a small comment.

File: packages/workspace/src/schematics/remove/lib/update-tsconfig.ts (hunk 1 above)
Comment (jcvalerio) [reply packages/workspace/src/schematics/remove/lib/update-tsconfig.ts]: Sure!!, changes were applied 👍 

The variable was renamed to keep the same style as
`const nxJson = readJsonInTree<NxJson>(tree, 'nx.json');`

**Comment by vsavkin** [comment]
Looks great! Thank you.

**Pull request merged**